elements/lisk-dpos/src/delegates_info.ts

```diff
@@ -69,21 +69,6 @@ export class DelegatesInfo {
 		return this._update(header, stateStore, { undo, delegateListRoundOffset });
 	}
 
-	public async undo(
-		header: BlockHeader,
-		stateStore: StateStore,
-		{ delegateListRoundOffset }: DPoSProcessingOptions,
```

Comment:
do we need `DPoSProcessingOptions` type that has `undo` field? maybe we can remove that too